scaleway/resource_redis_cluster.go

```diff
@@ -114,6 +105,75 @@ func resourceScalewayRedisCluster() *schema.Resource {
 					Type: schema.TypeString,
 				},
 			},
+			"private_network": {
+				Type:     schema.TypeList,
+				Optional: true,
+				//MaxItems:    1,
```

Comment:
Could be interesting to know what is the limit here.

Reply:
Turns out it's more complicated than I thought. According to the new messages about this issue, there are a lot of differences between standalone mode and cluster mode so I don't think I can write a constraint in the schema. From what I got :
- Cluster mode : you can link a single PvN at creation (with at least as many IPs as nodes) and can't edit nor delete it.
- Standalone mode : you can link several PvN and a PvN at the same time, you can edit/detach PvN, etc.

Reply:
I would suggest dropping the constraint in the schema and documenting those restrictions in the documentation of the resource.